src/interpreter_run.cc

```diff
@@ -1083,6 +1084,7 @@ Interpreter::Result Interpreter::run() {
 
       // GC might have taken place in object heap but local "method" is from program heap.
       PUSH(result);
+      process_->set_current_primitive_call(-1, -1);
```

Comment:
This also has to be done in the non-failure case (around line 1100 post change). You could also argue that you just throw away interesting information here, so maybe you shouldn't reset this and just keep track of the last primitive call?

Reply:
I think we want to clear the BCP again, as it has otherwise the potential to give us the wrong impression.
done.